src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/LoopingMessageProcessorServiceImplTest.java

```diff
@@ -26,6 +27,9 @@
     @Mock
     private MessageConsumer messageConsumer;
 
+    @Mock
+    private ConsumerDelayStrategy consumerDelayStrategy;
```

Comment:
you should now be able to verify that the dalyStrategy gets called

Reply:
Verifying